geonode/api/resourcebase_api.py

```diff
@@ -410,14 +410,15 @@ def get_search(self, request, **kwargs):
             objects = []
 
         object_list = {
-           "meta": {"limit": settings.CLIENT_RESULTS_LIMIT,
-                    "next": next_page,
-                    "offset": int(getattr(request.GET, 'offset', 0)),
-                    "previous": previous_page,
-                    "total_count": total_count,
-                    "facets": facets,
-                    },
-            'objects': map(lambda x: self.get_haystack_api_fields(x), objects),
+           "meta": {
+                "limit": settings.CLIENT_RESULTS_LIMIT,
```

Comment:
@Coop56 we have also this one in settings https://github.com/GeoNode/geonode/blob/master/geonode/settings.py#L783 which is for that specific purpose. CLIENT_RESULTS_LIMIT is actually the one that the js will use to limit the number of results returned in the api query.